spec/support/vcr_setup.rb

```diff
@@ -0,0 +1,8 @@
+require 'vcr'
+
+VCR.configure do |config|
+  config.cassette_library_dir = 'spec/vcr_cassettes'
+  config.hook_into :webmock
+  config.filter_sensitive_data('<google_api_key>') { ENV['GOOGLE_API_KEY'] }
```

Comment:
You could try using `ENV.fetch('sth')` instead of z`ENV['sth']`. Fetch throws an error when variable is missing while ENV['sth'] returns nil. I spent several hours once looking for solution to not working feature while the problem was missing variable. I wish I'd used fetch then 🙈

Reply:
thanks for advise ! <3